x-pack/plugins/alerting/server/plugin.ts

```diff
@@ -173,16 +173,6 @@ export class AlertingPlugin {
     muteAlertInstanceRoute(router, this.licenseState);
     unmuteAlertInstanceRoute(router, this.licenseState);
 
-    alertTypeRegistry.register({
```

Comment:
I'm a little confused as to why this was in this file in the first place!  Did it get left in from a previous PR, someone was using to test with?  Good find!